daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-architecture.md

```diff
@@ -173,6 +174,29 @@ Also, the Dapr Workflow engine requires that all instances of each workflow app
 
 Workflows don't control the specifics of how load is distributed across the cluster. For example, if a workflow schedules 10 activity tasks to run in parallel, all 10 tasks may run on as many as 10 different compute nodes or as few as a single compute node. The actual scale behavior is determined by the actor placement service, which manages the distribution of the actors that represent each of the workflow's tasks.
 
+## Workflow backend
+
+The workflow backend is responsible for orchestrating and preserving the state of workflows. At any given time, only one backend can be supported. The workflow backend can be configured as a component, similar to any other component in Dapr. Users are required to specify the type of workflow backend and provide the configuration specific to that backend.
+
+For instance, the following sample demonstrates how to define a SQLite backend component. Please note that as of now, SQLite backend is not supported. Dapr workflow currently supports only the actor backend by default, and users are not required to define an actor backend component to use it.
```

Comment:
Can we show a different example using a supported backend component, instead of SQLite?

Reply:
I will take out SQLite and not mention it becaused it is not supported as of now. I can add actor backend instead.